index.js

```diff
@@ -200,16 +200,24 @@ app.post('/checkin', (req, res) => {
             let currentDate = new Date(Date.now());
             // Adjust time to match US Timezone
             currentDate.setHours(currentDate.getHours() - 8);
-            currentDateString = currentDate.toDateString();
+            const currentDateString = currentDate.toDateString();
```

Comment:
I previously forgot to declare this variable lol, whoops 😅

![](https://media.giphy.com/media/14aUO0Mf7dWDXW/giphy.gif)